hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/om/ReconfigureOMCommands.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.ozone.admin.om;
+
+import org.apache.hadoop.hdds.cli.GenericCli;
+import org.apache.hadoop.hdds.cli.HddsVersionProvider;
+import org.apache.hadoop.hdds.cli.SubcommandWithParent;
+import org.kohsuke.MetaInfServices;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Model.CommandSpec;
+import picocli.CommandLine.Spec;
+
+import java.util.concurrent.Callable;
+
+/**
+ * Subcommand to group reconfigure OM related operations.
+ */
+@Command(
+    name = "reconfig",
+    description = "Dynamically reconfig OM without restarting server",
+    mixinStandardHelpOptions = true,
+    versionProvider = HddsVersionProvider.class,
+    subcommands = {
+        ReconfigureOMStartSubcommand.class, 
+        ReconfigureOMStatusSubcommand.class,
+        ReconfigureOMPropertiesSubcommand.class
+    })
+@MetaInfServices(SubcommandWithParent.class)
+public class ReconfigureOMCommands implements Callable<Void>,
```

Comment:
Please rename to be service-agnostic (`ReconfigureCommands`, `ReconfigureStartSubcommand`, etc.).